build-tools/packages/build-cli/src/library/commands/generateEntrypoints.ts

```diff
@@ -291,10 +323,19 @@ const generatedHeader: string = `/*!
 
 `;
 
+/**
+ * Generate "rollup" entrypoints for the given main entrypoint file.
+ *
+ * @param mainEntrypoint - path to main entrypoint file
+ * @param mapApiTagLevelToOutput - level oriented ApiTag to output file mapping
+ * @param log - logger
+ * @param separateBetaFromAlpha - if true, beta APIs will not be included in alpha outputs
+ */
 async function generateEntrypoints(
 	mainEntrypoint: string,
 	mapApiTagLevelToOutput: Map<ApiTag, ExportData>,
 	log: CommandLogger,
+	separateBetaFromAlpha: boolean,
```

Comment:
nit: in general we (I) try to keep the logger as the last parameter. Not a big deal though. The pattern started because the logger was often optional.

Reply:
yep. I think I'll keep it because after a while I'd like to remove that option altogether.